3rdparty/mshadow/mshadow/base.h

```diff
@@ -312,6 +338,11 @@ enum TypeFlag {
   kInt8  = 5,
   kInt64 = 6,
   kBool = 7,
+  kInt16 = 8,
+  kUint16 = 9,
+  kUint32 = 10,
+  kUint64 = 11,
```

Comment:
Why adding those additional types here? No operator supports them anyway, right?

Reply:
This is to align the definition with DLPack. Otherwise we have to preserve those numbers. Even through we don't use them currently, it's no harm to add them.